test/support/factory.ex

```diff
@@ -1036,6 +1038,21 @@ defmodule Trento.Factory do
     }
   end
 
+  def activity_log_entry_factory(attrs) do
```

Comment:
> What we could do is explicitly set inserted_at to DateTime.utc_now() in current activity_log_entry_factory and let the factories take care of overriding properties for us.

I thought it's better not to assume a default in mocked data, as it may cover the expected default in the schema. If we explicitly set the factory with a default `DateTime.utc_now()`, what if we change the schema not to have defaults for the field?

Reply:
if we think the approximation of the insertion in the db is not a proper implementation, we can add a different field like `occurred_on` and have the application generate it. I don't have anything against that.

With regards to this specific factory variation, the test passes even if we did not have it. Up to you whether keeping or removing.

Reply:
I think that was my misunderstanding of how factories work. I removed the factory, thanks.